### PR TITLE
Deploy to prod: Intro to Eth, pt 1 + 2

### PR DESCRIPTION
**Pull request by wolovim**

### CODE REVIEW

File: src/contexts/AppContext.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { type Project, type Fundamental } from "@/interfaces";
+import { type Project, type Fundamental, type EthIntro } from "@/interfaces";
```
Comment (Markkos89) [inline src/contexts/AppContext.tsx]: @wolovim why are we importing `EthIntro` here? it's not being used, right? :thinking: 

```ts
type EthIntro
```

File: src/contexts/AppContextProvider.tsx [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { AppContext } from "./AppContext";
 import {
   type Project,
   type Fundamental,
+  type EthIntro,
```
Comment (Markkos89) [inline src/contexts/AppContextProvider.tsx]: same here, type not being used?

File: src/interfaces/index.ts [hunk 3]
```diff
@@ -43,6 +43,13 @@ export interface Project {
   completed?: boolean;
 }
 
+export interface EthIntro {
+  path: string;
+  frontMatter: ProjectFrontMatter;
+  slug: string;
+  completed?: boolean;
+}
+
```
Comment (Markkos89) [inline src/interfaces/index.ts]: If this interface is not being used there is no need to have it, right?

Maybe I'm wrong.

this interface is the same as `Project` interface and should be common for all the project's lessons

**Review by Markkos89 — APPROVED** [review]
Approving here @wolovim !! :ok_hand: 

great work!!! :muscle: 

I left some question that can be addressed in other PR if it's needed as the interfaces mentioned are not being used in any place.

:rocket:

File: src/contexts/AppContext.tsx (hunk 1 above)
Comment (wolovim) [reply src/contexts/AppContext.tsx]: oh weird, i thought i deleted all these. i can add a quick PR to remove them first.

**Pull request merged**